sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -44,7 +44,8 @@
 
 #include "net/gnrc/ipv6.h"
 
-#define ENABLE_DEBUG        0
+#define ENABLE_DEBUG        \
```

Comment:
why doesn't it show any annotation here?

Reply:
Because there are so many warnings by `vera++` ^^. I removed it in a test commit I have here locally. I am currently debugging another thing however ;-).

Reply:
Done. Let's see if it annotates now :-).